build/ndo_build.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+#
```

Comment:
We can use single `build.sh` file to build both bins , for example the way we are doing in Maya repo ... again this can be done in subsequent PR's